roles/dhcp-server/templates/boxes.conf.j2

```diff
@@ -0,0 +1,15 @@
+# /etc/dnsmasq.d/boxes.conf
+# {{ ansible_managed }}
+
+dhcp-option-force=209,"partman-auto/disk=/dev/{{default_install_dev}} grub-installer/bootdev=/dev/{{default_install_dev}}"
+
+{% for box in boxes %}
+{% if box.comment is defined %}# {{box.hostname}} {{box.comment}}{% endif %}
```

Comment:
ah I get it now.. um.. you can use funky hostnames to force it into a mode that probably wont work.  it's a feature. :1st_place_medal: 

something like this - prefix the tags with h-

dhcp-host={{box.mac}},set:h-{{box.hostname}},{{box.hostname}}...
dhcp-option-force=tag:h-{{box.hostname}},209,"partman-auto/disk